index.js

```diff
@@ -1,4 +1,4 @@
-"use strict";
+// "use strict";
```

Comment:
It's not ok to comment all theses `"use strict";`, at least we remove them.
Did you encountered a problem ?